ksqldb-rest-app/src/main/java/io/confluent/ksql/api/server/DelimitedQueryStreamResponseWriter.java

```diff
@@ -54,7 +54,11 @@ public QueryStreamResponseWriter writeMetadata(final QueryResponseMetadata metaD
 
   @Override
   public QueryStreamResponseWriter writeRow(final GenericRow row) {
-    response.write(ServerUtils.serializeObject(row.values()).appendString("\n"));
+    if (row == null) {
+      response.write("\n");
```

Comment:
Why start a new line if `row` is null? If we have less traffic would that cause a lot of new lines?